src/Utils/QtiMarshallerUtil.php

```diff
@@ -93,4 +100,31 @@ public static function marshallValidQti(QtiComponent $component)
 
         return $dom->saveXML($node);
     }
+
+    public function marshallObjectData(QtiComponent $component)
+    {
+        $result = '';
+        $class = new \ReflectionClass(get_class($component));
+        if (property_exists($component, 'data') && property_exists($component, 'type')) {
+            $property = $class->getProperty('data');
+            $property->setAccessible(true);
+            $property_type = $class->getProperty('type');
```

Comment:
Style: Lower camel case for variable name `$property_type`.

Reply:
Done